skyportal/tests/frontend/test_frontpage.py

```diff
@@ -129,7 +129,7 @@ def test_source_filtering_and_pagination(driver, user, public_group, upload_data
         time.sleep(1)
         assert not next_button.is_enabled()
 
-
+@pytest.mark.flaky(reruns=2)
```

Comment:
@profjsb this will fail if pytest isn't imported above

Reply:
Fixed